draft-ietf-suit-manifest-03.md

```diff
@@ -108,12 +108,12 @@ Distributing firmware updates to diverse devices with diverse trust anchors in a
 The various constraints on IoT devices creates a broad set of use-case requirements. For example, devices with:
 
 * limited processing power and storage may require a simple representation of metadata.
-* bandwidth constraints may require firmware compression or partial update support.
+* bandwidth constraints may require firmware compression or differential update support.
```

Comment:
The original wording "partial" was intended to mean module update without compression. This is part of the basis for dependencies and multiple payloads. Does that seem valuable to mention?

Reply:
I guess the problem is that we did not define what a partial update is. We also do not define what a module update without compression is. Maybe we should do that...